src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -256,7 +256,7 @@ TEST_CASE("SimpleDropoutLayerTest", "[ANNLayerTest]")
   // Test the Forward function.
   arma::mat output;
   module.Forward(input, output);
-  REQUIRE(arma::as_scalar(arma::abs(arma::mean(output) - (1 - p))) <= 0.05);
+  REQUIRE(arma::as_scalar(arma::abs(arma::mean(output) - (1 - p))) <= 0.05); 
```

Comment:
```suggestion
  REQUIRE(arma::as_scalar(arma::abs(arma::mean(output) - (1 - p))) <= 0.05);
```
Minor style fix :)

Reply:
:(

Reply:
Please do remove trailing whitespace.

Reply:
This check is also an absolute value check---maybe we should rework it too to be a relative value check?